mxnet_model_service.py

```diff
@@ -0,0 +1,95 @@
+import mxnet as mx
+import numpy as np
+import tarfile
+import json
+import os
+
+from mxnet.gluon.utils import download
+from mxnet.io import DataBatch
+from model_service import SingleNodeModel, MultiNodesModel
+
+URL_PREFIX = ('http://', 'https://', 's3://')
+MODEL_DIR = 'models/'
+CONFIG_FILE = 'config.json'
```

Comment:
I prefer config file name is not hardcoded. 
It can be passed as a parameter with default config.json
There is also a problem, if two models zip in the same folder, config file might be overwritten.

Reply:
The problem is that we may not want to add more CML arguments at this time. We can just pop up an error message reminding user of this.